src/components/schedule-page/table/schedule/schedule-parts/base-cell/right-click-popper-content.component.tsx

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import React from "react";
+import { TranslationHelper } from "../../../../../../helpers/translations.helper";
+import { shifts } from "../../../../../../common-models/shift-info.model";
+import { MdClose } from "react-icons/md";
+
+export interface DisplayDetailsOptions {
+  index: number;
+  day: number;
+  month: number;
+  year: number;
+  rowIndex: number;
+  shift: string;
+  sectionKey: string;
+  close: () => void;
+}
+
+export function RightClickPopperContent(props: DisplayDetailsOptions): JSX.Element {
+  const { index, day, month, year, rowIndex, shift, sectionKey, close } = props;
+  let displayedYear = year;
+  let monthName = `${TranslationHelper.polishMonthsGenetivus[month]}`;
+  if (index < day) {
+    if (month > 0) {
+      monthName = `${TranslationHelper.polishMonthsGenetivus[month - 1]}`;
+    } else {
+      monthName = `${TranslationHelper.polishMonthsGenetivus[month + 11]}`;
+      displayedYear = year - 1;
+    }
+  }
+  let shiftcode = shift;
+  if (shiftcode === "") shiftcode = "W";
+  const foundShift = shifts[shiftcode];
+
+  return (
+    <div className="align-to-the-left">
+      <div className="exit-button" onClick={close}>
+        <MdClose />
+      </div>
+      <div style={{ fontWeight: "lighter" }}>
+        {day} {monthName} {displayedYear} r.
+      </div>
+      <div>
+        Pracownik o indeksie {rowIndex} w sekcji {sectionKey}
```

Comment:
Tutaj powinno być imię i nazwisko pracownika. 
Można to dostać ze stanu za pomocą indeksu w taki sposób:
```ts
const worker = useSelector(state => Object.keys(state.persistentSchedule.present.shifts)[index])
```

Reply:
Próbowałam, ale to nie za bardzo działa ![nurse](https://user-images.githubusercontent.com/48785655/105546588-db42c700-5cfd-11eb-8842-8c7de8975d7f.png)

Reply:
ok, tylko jest problem:
![nurse](https://user-images.githubusercontent.com/48785655/105549278-109ce400-5d01-11eb-8bd8-9d1788cbcdc0.png)

Reply:
Mi się wydaje, że wiem w czym jest problem. Powiedziałem ci wcześniej  głupotę i zamiast `index` powinno być `rowIndex`.
Czy mogłabyś to spróbować?
Wiem, te intuicyjne nazwy XD

Reply:
tak, do tego już doszłam XD screen powyżej jest właśnie z `rowIndex`em, wcześniej to się w ogóle działo :stuck_out_tongue_closed_eyes: